examples/nerdctl-ipfs-registry-kubernetes/nerdctl-ipfs-registry.yaml

```diff
@@ -0,0 +1,287 @@
+# Example YAML of IPFS-based node-to-node image sharing
+
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: ipfs-bootstrap
+spec:
+  selector:
+    matchLabels:
+      app: ipfs-bootstrap
+  template:
+    metadata:
+      labels:
+        app: ipfs-bootstrap
+    spec:
+      initContainers:
+        - name: configure-ipfs
+          image: "ghcr.io/stargz-containers/ipfs/go-ipfs:v0.11.0"
+          command: ["sh", "/custom/configure-ipfs.sh"]
```

Comment:
You can probably omit `sh`

Reply:
This is needed to prevent permission error.